src/Component/Configuration/Loader/SassLoader.php

```diff
@@ -0,0 +1,72 @@
+<?php
+namespace Hostnet\Component\Webpack\Configuration\Loader;
+
+use Hostnet\Component\Webpack\Configuration\CodeBlock;
+use Hostnet\Component\Webpack\Configuration\ConfigExtensionInterface;
+use Symfony\Component\Config\Definition\Builder\NodeBuilder;
+
+final class SassLoader implements LoaderInterface, ConfigExtensionInterface
+{
+    /**
+     * @var array
+     */
+    private $config;
+
+    /**
+     * @param array $config
+     */
+    public function __construct(array $config = [])
+    {
+        $this->config = $config;
+    }
+
+    /** {@inheritdoc} */
+    public static function applyConfiguration(NodeBuilder $node_builder)
+    {
+        $node_builder
+            ->arrayNode('sass')
+                ->canBeDisabled()
+                ->addDefaultsIfNotSet()
+                ->children()
+                    ->booleanNode('all_chunks')->defaultTrue()->end()
+                    ->scalarNode('filename')->defaultNull()->end()
+                ->end()
+            ->end();
+    }
+
+    /** {@inheritdoc} */
+    public function getCodeBlocks()
+    {
+        $config      = $this->config['loaders']['sass'];
+        $code_blocks = [];
+
+        if (! $config['enabled']) {
+            return [new CodeBlock()];
+        }
+
+        if (empty($config['filename'])) {
+            // If the filename is not set, apply inline style tags.
+            $code_blocks[] = (new CodeBlock())->set(CodeBlock::LOADER, '{ test: /\.scss$/, loader: \'style!css!sass\' }');
+        } else {
+            // If a filename is set, apply the ExtractTextPlugin
+            $fn            = 'fn_extract_text_plugin_sass';
+            $code_blocks[] = (new CodeBlock())
+                ->set(CodeBlock::HEADER, 'var '.$fn.' = require("extract-text-webpack-plugin");')
```

Comment:
Some minor code style inconsistencies; Can you add spaces around the dots for concatenation?